src/DOM/File/Blob.purs

```diff
@@ -0,0 +1,22 @@
+module DOM.File.Blob
+  ( type_
+  , size
+  ) where
+
+import Prelude ((==))
+import Data.Maybe (Maybe(..))
+
+import DOM.File.Types (Blob())
+
+foreign import typeImpl :: Blob -> String
+
+-- | A string indicating the MIME type of the data contained in the `Blob`.
+-- | Returns `Nothing` if the type is unknown.
+type_ :: Blob -> Maybe String
+type_ blob = let blobType = typeImpl blob
+             in if blobType == ""
+                then Nothing
+                else Just blobType
```

Comment:
Ok done, and published: https://github.com/purescript-contrib/purescript-media-types

v0.1.0 for now, as everything will need bumping when we do the core libraries for 0.8.